ui/js/models/resultsets_store.js

```diff
@@ -290,7 +290,7 @@ treeherder.factory('ThResultSetStore', [
             }
         };
 
-        var getAllShownJobs = function (repoName, spaceRemaining, maxSize, resultsetId) {
+        var getAllShownJobs = function (repoName, spaceRemaining, errorMessage, resultsetId) {
```

Comment:
I'm not sure errorMessage is needed here? maxSize was the count passed to the error message below, whereas errorMessage is the entire string. But it raises a question, which is it might be less brittle if the unified message `.constant` in treeherder.js in this change, was a variable that passed in the 'max' number? So if someone changed the maxSize allowed in the future, the error message would reflect it automatically, rather than be a static message and not match it.

Granted we haven't changed the max size since the early development of treeherder. But with the above approch we'd never need to maintain the error message.

Reply:
For posterity, this was the thing I was wondering we could hook the error message to and glue it to the end of the constant. So if/when someone changes the limit, the error message will automatically update and we don't have to touch it again.

eg. constant > "Max pinboard size reached." + "Maximum allowed " + maxNumPinned
https://github.com/mozilla/treeherder/blob/7233dfd46f63c8d478c1ddbdebe608023d3d32e5/ui/js/services/pinboard.js#L180

Reply:
It's a fair point, but maybe we can address it in a later PR if someone feels like it.  This seems OK for now.  As you say, I don't know that we'll be changing this number any time soon.  :)